apps/app_manager.py

```diff
@@ -1,11 +1,17 @@
-import traceback
+from __future__ import print_function
```

Comment:
You don't actually need this to use print() as a function in Python 2.7 and further =)

Reply:
You're right fixed it.